### DISCUSSION TITLE
Vercel should ignore test files in api folder

### DISCUSSION THREAD

**Discussion by jakeleventhal** [vercel/vercel#9896] [General]
### Description

https://vercel.com/docs/concepts/functions/serverless-functions#adding-utility-files-to-the-/api-directory
The docs specify:
>Files that start with an underscore, _, are ignored
Files that start with . are ignored
Files that end with .d.ts are ignored

.test.ts files should also be ignored. a common pattern is to add test files in the same location as the src files. At the very least, there should be an option to specify a glob to ignore

**Comment by amyegan** [comment]
Duplicate of https://github.com/vercel/community/discussions/2286
